### PR TITLE
updated just the model, not the routes/api.

### PR DESCRIPTION
**Pull request by aakarshgarg23**

### CODE REVIEW

File: back-end/models/Table.js [hunk 1]
```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
 
 // Create Schema
 const TableSchema = new Schema({
-  location: {
+
+  ID:{
```
Comment (josephmaxwellhyland) [inline back-end/models/Table.js:8]: we might not actually need an ID - I think we can use the built-in auto-id from mongoose.

https://mongoosejs.com/docs/guide.html#_id

What do you think @brebory ?

File: back-end/models/Table.js [hunk 2]
```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
...
-    default: 'submitted'
+    required: false
   },
-  tableType: {
+  zipcode: {
     type: String,
-    enum: ['fixed-height', 'adjustable', 'portable'],
-    required: true
+    required: false,
+    max:5
```
Comment (josephmaxwellhyland) [inline back-end/models/Table.js:33]: Is there a way to do a minimum of 5 too? i think all zip codes should be exactly 5 digits

File: back-end/models/Table.js [hunk 3]
```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
...
+    type: String,
+    required: false
   },
   restroomType: {
     type: String,
     enum: ['men', 'women', 'family', 'other'],
+    required: false
+  },
+  coordinateLocation: {
+    type: Array,
```
Comment (josephmaxwellhyland) [inline back-end/models/Table.js:46]: I believe we want this to be type Point: https://www.mongodb.com/docs/manual/reference/geojson/

**Review by josephmaxwellhyland — COMMENTED** [review]
Just a couple of questions - @brebory @JasonOsborne and J (who I can't tag for some reason) let me know if you have thoughts!

File: back-end/models/Table.js (hunk 1 above)
Comment (brebory) [reply back-end/models/Table.js:8]: Yep, using the built-in id makes sense to me.

**Review by josephmaxwellhyland — APPROVED** [review]
Looks good to me!

File: back-end/models/Table.js (hunk 2 above)
Comment (josephmaxwellhyland) [reply back-end/models/Table.js:26]: I think it's fine without that

**Pull request merged**